newIDE/app/src/SceneEditor/index.js

```diff
@@ -887,6 +896,10 @@ export default class SceneEditor extends React.Component<Props, State> {
     if (this._objectsList) this._objectsList.forceUpdateList();
   };
 
+  forceUpdateLayersList = () => {
+    if (this._layersList) this._layersList.onLayerModified();
+  };
+
```

Comment:
@4ian I'm not sure if this is good or not :/ I believe the `LayersList` wasn't updating, after adding and removing the layer twice, it was in a state where there was no lighting layer and the button was disabled as well. Now, force updating does solve the problem but I'm not sure if it's the right way :/

Reply:
Let's put back `_onLayerModified` "private" and use `forceUpdate` instead.

Force updating these components showing virtualized list is fine - maybe in the future we'll rework them to avoid this.